src/dotnet-install.sh

```diff
@@ -961,9 +961,9 @@ get_remote_file_size() {
     local zip_uri="$1"
 
     if machine_has "curl"; then
-        file_size=$(curl -sI  "$zip_uri" | grep -i content-length | awk '{print $2}')
+        file_size=$(curl -sI  "$zip_uri" | grep -i content-length | awk '{ num = $2 + 0; print num }')
     elif machine_has "wget"; then
-        file_size=$(wget --server-response -O /dev/null "$zip_uri" 2>&1 | grep -i '^Content-Length:' | awk '{print $2}')
+        file_size=$(wget --server-response -O /dev/null "$zip_uri" 2>&1 | grep -i '^Content-Length:' | awk '{ num = $2 + 0; print num }')
```

Comment:
I am testing this locally and the `wget` branch seems to be broken. Can you please confirm that this command works for you? My `wget` prints headers indented so the `grep` regex won't match it.

```
~$ wget --server-response -O /dev/null "http:some.random.url"
--2023-09-15 16:44:40--  http://some.random.url
Connecting to 192.168.1.9:80... connected.
HTTP request sent, awaiting response...
  HTTP/1.1 200 OK
  Date: Fri, 15 Sep 2023 14:44:40 GMT
  Server: Apache/2.4.41 (Ubuntu)
  Last-Modified: Fri, 15 Sep 2023 09:36:51 GMT
  ETag: "2e99060-60562883e92c0"
  Accept-Ranges: bytes
  Content-Length: 48861280
  Keep-Alive: timeout=5, max=100
  Connection: Keep-Alive
  Content-Type: application/vnd.android.package-archive
Length: 48861280 (47M) [application/vnd.android.package-archive]
Saving to: '/dev/null'
```

Reply:
For wget we don't need "^", it's a good catch!
![image](https://github.com/dotnet/install-scripts/assets/95473390/f3421a27-4827-496a-bc7c-893dd2e3dd70)

Reply:
curl works as expected
![image](https://github.com/dotnet/install-scripts/assets/95473390/63f5cbdc-4bc3-4536-b3d3-cb47e3e5cbb7)